frontend/app/controllers/Offer.scala

```diff
@@ -2,17 +2,28 @@ package controllers
 
 import actions.Functions._
 import configuration.Config
+import model.MembersOnlyContentExtractor
 import play.api.mvc.Controller
+import services.GuardianContentService
+
 
 trait Offer extends Controller {
   val permanentStaffGroups = Config.staffAuthorisedEmailGroups
   val AuthorisedStaff = GoogleAuthenticatedStaffAction andThen isInAuthorisedGroupGoogleAuthReq(
     permanentStaffGroups, views.html.fragments.oauth.staffWrongGroup())
+  val contentApiService = GuardianContentService
 
   // TODO move this to CachedAction once this work is ready to go into the wild
   def subscriber = AuthorisedStaff { implicit request =>
     Ok(views.html.offer.subscriber())
   }
+
+  // TODO FYI this controller was originally meant for the subscribers landing page - maybe we need a rethink on the names
+  // TODO of these controllers as I feel these methods don't belong together
+  def offersAndComps = CachedAction { implicit request =>
```

Comment:
More of a reminder that the subscriber work and offers and Comps work doesn't feel right sat here together. 
I feel there maybe a bigger question about urls also related to this. I just created the url for subscriber so maybe @davidmcdowell needs input on that and that may filter down to the controller and where these two sit.